src/docs/product/performance/performance-overhead.mdx

```diff
@@ -32,3 +32,10 @@ Each SDK has limits on the data structures used to record all the above data. If
 ### Backpressure Management
 
 The Python SDK, starting with version `1.30.0`, manages backpressure internally by reducing the dynamic sampling rate when the system is in an unhealthy state. This ensures better system stability and lower SDK overhead under heavy load. We will gradually roll this out to other backend languages in the future.
+
+### Hosting Environment
+
+Some hosting environments have mechanisms in place that will have a negative influence on SDK overhead:
+
+- AWS Lambdas completely freeze any code execution when a response is sent. This means that the SDKs generally will block the response from being finished until it has flushed all of the recorded telemetry data to Sentry. This might cause non-trivial overhead in some situations. Here, it might be beneficial to run a Sentry Relay instance physically close to your lambdas to decrease latency.
```

Comment:
```suggestion
- AWS Lambda completely freezes any code execution when a response is sent. To keep code running, the SDKs generally block the response from being finished until it has flushed all of the recorded telemetry data to Sentry. This might cause non-trivial overhead in some situations. Here, it might be beneficial to run a Sentry Relay instance physically close to your lambdas to decrease latency.
```

I think "physically close" is a little confusing here. Do you mean in the same region or availability zone, or something else?

Reply:
Yes. But I don't really want to write "same region" or same "availability zone" because having servers that run physically close to eachother will generally have lower latency. They don't necessarily need to be in the same region and "availability zones" are technically something unrelated to locality.